doc/classes/CubemapArray.xml

```diff
@@ -1,8 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CubemapArray" inherits="TextureLayered" version="4.0">
 	<brief_description>
+		A [TextureLayered] of type [code]enum TextureLayeredType.TEXTURE_LAYERED_CUBEMAP_ARRAY[/code] which uses 6, or a multiple of 6, layers.
```

Comment:
This information should be included below. The brief description is mostly useful as the first bit of information a user sees in the tooltip. It should be something that a user who reads "CubemapArray" and has no idea what that means can get a quick understanding. 

Try something like "Composite texture type made up of an array of Cubemaps"

Reply:
I'll go with `A single texture resource which consists of multiple [Cubemap]s.`.

"Composite" is indeed a good word, but it's also rarely used. I'm not a native speaker and I still remember the first few times I encountered it. Had to look it up again and again, as it shows up so rarely...